packages/edge-login-ui-rn/src/components/themed/ThemedButtons.js

```diff
@@ -69,7 +69,7 @@ export function PrimaryButton(props: Props) {
       onPress={onPress}
     >
       {label != null ? <Text style={styles.primaryText}>{label}</Text> : null}
-      {spinner != null ? (
```

Comment:
but this current solution is really strange. I need something like this
```
<SecondaryButton spinner={isLoading} />
```
where `isLoading` is boolean.
But here to show the spinner I need to pass undefined or null. Whaaat?

Reply:
If a value is a boolean, (`true` or `false`), you can use it the normal way. So, `<SecondaryButton spinner={isLoading} />` is great! But, if a value is `SomeThing | undefined`, you need to do the `!= null` check.

Reply:
something like `spinner === true` will be good?
or, maybe you can provide your short solution?
because using just `spinner != null` confusing when you trying pass the boolean false value and see spinner on button